test/auro-accordion-group.test.js

```diff
@@ -25,12 +26,42 @@ describe('auro-accordion-group', () => {
         </auro-accordion>
       </auro-accordion-group>
     `);
-    
     const accordions = el.querySelectorAll('auro-accordion');
     const button = accordions[0].shadowRoot.querySelector('button');
+
     button.click();
 
     expect(accordions[0].expanded).to.be.true;
     expect(accordions[1].expanded).to.be.false;
-  })
+  });
+
+  it('reacts to slot changes', async () => {
```

Comment:
Given that this may become a boilerplate thing, it would be nice to template this test if possible.